case-app/angular-library
# Feature/arrow

### What does this PR do ?
This PR fix design issues.
- Set a higher z position of flash-message
- Change a z position of tooltip in the header
- Set a min width to dots 

### Issues closed
[case-app/#132](https://github.com/case-app/case/issues/132)
[case-app/#137](https://github.com/case-app/case/issues/137)

### How to test it ?
To see the flash message, you need to run a project with a form in a modal dialog. once on the modal dialog, just try to save a form with an error on any field. The flash-message should appear behind the modal layer.

To test the tooltip position, hover a top-menu-quick-link. The tooltip should appear above the notification number.
